examples/01a-Getting-started.ipynb

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
<p>We can use a utils function and do "Categorify" other preprocessing under the hood and store it to parquet. (If we want)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>works without any additional preprocessing so in the interest of keeping it simple and not dragging reader's attention from the dataloader would just keep the data as is</p>